hazelcast/src/test/java/com/hazelcast/map/query/QueryIndexingTest.java

```diff
@@ -134,41 +122,10 @@ private static Config newConfig(final Map<Integer, Employee> employees) {
             } else {
                 val = new Employee(i, "name" + i, "city" + i, 0, true, i);
             }
-            employees.put(i, Mockito.spy(val));
+            Employee spy = MockUtil.serializableSpy(Employee.class, val);
```

Comment:
It might not be obvious from this diff but if you have a look at the whole file and `assertGettersCalledNTimes` method I think its easier to see what's going on. I'm more used to JMockit than Mockito, learning it myself.